frontend/app/actions/CommonActions.scala

```diff
@@ -121,9 +121,9 @@ abstract class OAuthActions(parser: BodyParser[AnyContent], implicit val executi
 
   import commonActions.NoCacheAction
   //routes
-  private val loginTarget = routes.OAuth.loginAction()
```

Comment:
I think this change should be done the other way aaround - i.e. you woudl need to change it in the routes file instead. 
This is because it's using some eta expansion thing that's deprecated in scala 2.13 and will be removed in scala 3

Reply:
I know this because I fixed them all in support-frontend, then noticed all the scala 2.3 warnings, so fixed them, which then broke all these again.

Reply:
see https://github.com/guardian/support-frontend/pull/3122/files#r653618017

Reply:
Thanks, I've updated as you suggest